LayoutTests/fast/frames/lots-of-objects.html

```diff
@@ -1,35 +1,49 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!DOCTYPE html>
 <html>
 <head>
 <title>Lots of objects</title>
-<script type="text/javascript">
-  var maxNumberOfFrames = 1000;
+<script>
+var maxNumberOfFrames = 1000;
 
-  if (window.testRunner)
+if (window.testRunner)
     testRunner.dumpAsText();
 
-  var createdObjects = 0;
-  function count() {
-    createdObjects++;
+function dumpResult()
+{
+    let objects = document.getElementsByTagName("object");
+    let objectCount = objects.length;
+    var result = "";
 
-    if (createdObjects == maxNumberOfFrames) {
-      var str = "Created precisely " + maxNumberOfFrames + " objects.<br>PASS";
-      document.getElementById("status").innerHTML = str;
-    } else if (createdObjects > maxNumberOfFrames) {
-      var str = "Created " + createdObjects + " objects (max: " + maxNumberOfFrames + ").<br>FAIL";
-      document.getElementById("status").innerHTML = str;
-    }
-  }
+    if (objectCount != maxNumberOfFrames + 1)
+        result = `Created an invalid number of objects (${objectCount}).<br>FAIL`;
+    else if (objects[objectCount - 1].contentDocument)
+        result = `Created ${objectCount} objects (max: ${maxNumberOfFrames}).<br>FAIL`;
+    else
+        result = `Created precisely ${maxNumberOfFrames} objects.<br>PASS`;
```

Comment:
Nit Would it be cleaner to use the helper functions for `testPassed(...)` and `testFailed(...)` rather than manually modifying the status div's inner HTML?